apollo-router/src/plugins/demand_control/mod.rs

```diff
@@ -342,16 +343,19 @@ impl Plugin for DemandControl {
                     // On the request path we need to check for estimates, checkpoint is used to do this, short-circuiting the request if it's too expensive.
                     Ok(match strategy.on_execution_request(&req) {
                         Ok(_) => ControlFlow::Continue(req),
-                        Err(err) => ControlFlow::Break(
-                            execution::Response::builder()
-                                .errors(
-                                    err.into_graphql_errors()
-                                        .expect("must be able to convert to graphql error"),
-                                )
-                                .context(req.context.clone())
-                                .build()
-                                .expect("Must be able to build response"),
-                        ),
+                        Err(err) => {
+                            emit_error_event(err.code(), "Demand control execution error");
```

Comment:
We'll now emit a log for every request that is rejected. I think this could/will cause a flood of logs to the users' APM. I don't think we can go with this as is. I'm going to do a bit of reading around what the latest recommendations are around logging and comment further.

Reply:
By log do you mean to standard out? Because the changes to `fmt_layer.rs` will prevent that.

Reply:
Ahh I see!